src/EditorFeatures/CSharpTest/CodeActions/IntroduceVariable/IntroduceVariableTests.cs

```diff
@@ -1353,8 +1353,8 @@ class Program
     static void Main(string[] args)
     {
         var d = new Dictionary<string, Exception>();
-        var {|Rename:exception|} = new Exception();
-        d.Add(""a"", exception);
+        var {|Rename:value|} = new Exception();
+        d.Add(""a"", value);
```

Comment:
I think there are 3 options here:
1. Always prefer parameter name  - that's what's implemented here.
2. Have heuristics about when to prefer parameter name, based on either the type of argument, or the type of receiving construct.
3. Have this be a last chance, to use if we don't find a better name, and would otherwise fall back to `v`/`v1`, etc.

I'm not sure whether I prefer 1 or 3, but tentatively I think 1.  2 could work, but I'm not sure what the heuristic would be .  I don't think it's as simple as just "is a generic type".  I really think it's more like "is an element of a collection such that the parameter name doesn't really matter".  Even in the case of something like a dictionary, extracting and getting `key` and `value` *does* seem useful.